kura/org.eclipse.kura.db.sqlite.provider/src/main/java/org/eclipse/kura/internal/db/sqlite/provider/SqliteWireRecordStoreImpl.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Eurotech and/or its affiliates and others
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.internal.db.sqlite.provider;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.eclipse.kura.KuraStoreException;
+import org.eclipse.kura.type.BooleanValue;
+import org.eclipse.kura.type.ByteArrayValue;
+import org.eclipse.kura.type.DoubleValue;
+import org.eclipse.kura.type.FloatValue;
+import org.eclipse.kura.type.IntegerValue;
+import org.eclipse.kura.type.LongValue;
+import org.eclipse.kura.type.StringValue;
+import org.eclipse.kura.type.TypedValue;
+import org.eclipse.kura.util.jdbc.ConnectionProvider;
+import org.eclipse.kura.util.wire.store.SqlWireRecordStoreHelper;
+import org.eclipse.kura.util.wire.store.SqlWireRecordStoreQueries;
+import org.eclipse.kura.wire.WireRecord;
+import org.eclipse.kura.wire.store.provider.WireRecordStore;
+
+@SuppressWarnings("restriction")
+public class SqliteWireRecordStoreImpl implements WireRecordStore {
+
+    private static final Map<Class<? extends TypedValue<?>>, String> TYPE_MAPPING = buildTypeMapping();
+
+    private final SqlWireRecordStoreHelper helper;
+
+    public SqliteWireRecordStoreImpl(final ConnectionProvider provider, final String tableName)
+            throws KuraStoreException {
+        final String sanitizedTableName = sanitizeSqlTableAndColumnName(tableName);
+
+        final SqlWireRecordStoreQueries queries = SqlWireRecordStoreQueries.builder()
+                .withSqlAddColumn("ALTER TABLE " + sanitizedTableName + " ADD COLUMN {0} {1};")
+                .withSqlCreateTable("CREATE TABLE IF NOT EXISTS " + sanitizedTableName
```

Comment:
Is there a particular reason for using `BIGINT` instead of `DATETIME` as the timestamp type?

Reply:
I have kept it in this way for consistency with the old DbStore and H2DbWireRecordStore implementations. I don't remember the reason for having the timestamp as an integer.